locust/launchdarkly_locust.py

```diff
@@ -11,21 +11,20 @@
 from locust_feature_requester import FeatureRequesterImpl as LocustServerFeatureRequester
 from locust_streaming import StreamingUpdateProcessor as LocustStreamingProcessor
 
-
-class LaunchDarklyLocust(Locust):
+class LaunchDarklyLocust(HttpUser):
     sdk_key = os.environ.get('LAUNCHDARKLY_SDK_KEY')
     default_user = {"anonymous": True, "key": "anonymous"}
     config_class = Config
     client_class = LDClient
     base_uri = os.environ.get('LAUNCHDARKLY_BASE_URI')
     events_uri = os.environ.get('LAUNCHDARKLY_EVENTS_URI')
     stream_uri = os.environ.get('LAUNCHDARKLY_STREAM_URI')
-    event_processor_class = LocustEventDispatcher
-    feature_requester_class = LocustServerFeatureRequester
-    update_processor_class = LocustStreamingProcessor
 
     def __init__(self, *args, **kwargs):
         super(LaunchDarklyLocust, self).__init__(*args, **kwargs)
+        self.update_processor_class = lambda config, store, ready: LocustStreamingProcessor(self.environment, config, store, ready)
```

Comment:
Well, this is a heavily edited version of the Python SDK code, but I believe the SDK works the same in this regard— yes it is treating it as a factory, but the property has "class" in the name because we originally figured we would just put a class reference in there, and a class _is_ a factory as far as Python is concerned (i.e. the class object can be called as a constructor).